engineer.tex

```diff
@@ -150,9 +150,9 @@ \chapter{Wymagania wszelakie}
 \end{requirementstab}
 
 
-\printbibliography[title={Bibliografia}]
+\printbibliography[title={Bibliografia}, heading=bibintoc]
```

Comment:
The bibliography in or not in table of contents is disputable. I will accept it for now.